nncf/quantization/algorithms/weight_compression/scale_estimation.py

```diff
@@ -418,7 +418,7 @@ def estimate_scales(weight: Tensor, target: Tensor, zero_mask: Tensor, importanc
     :param importance: The importance values tensor.
     :return: The estimated scales
     """
-    ideal_scale = fns.abs(weight) / (fns.abs(target) + zero_mask)
+    ideal_scale = fns.abs(weight) * fns.reciprocal(fns.abs(target) + zero_mask)
```

Comment:
How did you prove that this approximation is better for `cw= w*(1/scale) + zp`?

Reply:
With this change, I get `0.84391` similarity for `tinyllama_data_aware_gptq_scale_estimation_stateful` conformance test. Without it I got `0.8188` similarity.

However, PPL on phi3 model shows contradicting results, i.e. the metric is better without this change. Please see the updated PR description.

Because of the facts this change is not required for the alignment of compression methods for #2727, and division via reciprocal is in general less accurate, I propose to not to change it here.